src/OpenLoco/Map/BuildingTile.cpp

```diff
@@ -244,4 +263,15 @@ namespace OpenLoco::Map
         }
         return true;
     }
+
+    void registerHooks()
```

Comment:
If you move the function to tilemanager then you just add this hook into the already existing registerHooks.